config/frr-with-webhook/clusterrole-permissions.yaml

```diff
@@ -0,0 +1,61 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  labels:
+    app: metallb
+  name: metallb-system:controller
+rules:
+  - apiGroups:
```

Comment:
Would it make sense to patch with the ones needed for the webhooks and relying on the base for the others?